contribs/gmf/src/directives/search.js

```diff
@@ -965,7 +965,7 @@ gmf.SearchController.prototype.fulltextsearch_ = function(query, resultIndex, op
         const feature = format.readFeature(data.features[resultIndex - 1]);
         this.featureOverlay_.addFeature(feature);
         const fitOptions = /** @type {olx.view.FitOptions} */ ({});
-        if (opt_zoom) {
+        if (typeof opt_zoom !== 'undefined') {
```

Comment:
can be simplified to `if (opt_zoom !== undefined) {`

Reply:
+1